CONTRIBUTING.md

```diff
@@ -180,6 +180,64 @@ See [Translating](#translating) for more details.
 
 There is a dedicated [`apache-superset` tag](https://stackoverflow.com/questions/tagged/apache-superset) on [StackOverflow](https://stackoverflow.com/). Please use it when asking questions.
 
+## Types of Contributors
+
+Following Apache Software Foundation's project governance,
+Apache Superset has the following types of contributors
```

Comment:
@dpgaspar could we remove the forced newline here (and elsewhere). The text will wrap when necessary.

Reply:
ok, done that